<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>personalCenter</class>
 <widget class="QWidget" name="personalCenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="register_lab">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>个人中心</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="3,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>姓名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_name">
           <property name="styleSheet">
            <string notr="true">QLineEdit{
    background-color:rgba(0,0,0,0);
}</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>学号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_num">
           <property name="styleSheet">
            <string notr="true">QLineEdit{
    background-color:rgba(0,0,0,0);
}</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,3">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>联系方式</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_tel">
           <property name="styleSheet">
            <string notr="true">QLineEdit{
    background-color:rgba(0,0,0,0);
}</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,3">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>社团名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_clubname">
           <property name="styleSheet">
            <string notr="true">QLineEdit{
    background-color:rgba(0,0,0,0);
}</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,3">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>角色</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_role">
           <property name="styleSheet">
            <string notr="true">QLineEdit{
    background-color:rgba(0,0,0,0);
}</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,3">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>班级</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_class">
           <property name="styleSheet">
            <string notr="true">QLineEdit{
    background-color:rgba(0,0,0,0);
}</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,3">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>系别</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_type">
           <property name="styleSheet">
            <string notr="true">QLineEdit{
    background-color:rgba(0,0,0,0);
}</string>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,1">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{
    background-color:rgba(0,0,0,0);
}</string>
             </property>
             <property name="text">
              <string>年龄</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinbox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>性别</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <item>
              <property name="text">
               <string>男</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>女</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,3">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>加入时间</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="styleSheet">
            <string notr="true">QLineEdit{
    background-color:rgba(0,0,0,0);
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1">
       <item>
        <widget class="QLabel" name="label_picture">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1,1,1,1">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_picture">
       <property name="styleSheet">
        <string notr="true">QPushButton{
    background-color:rgba(0,0,0,0);
}</string>
       </property>
       <property name="text">
        <string>修改头像</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="yes_btn_2">
       <property name="styleSheet">
        <string notr="true">QPushButton{
    background-color:rgba(0,0,0,0);
}</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="no_btn_2">
       <property name="styleSheet">
        <string notr="true">QPushButton{
    background-color:rgba(0,0,0,0);
}</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
